test/integration/sanity_test.go

```diff
@@ -0,0 +1,137 @@
+// Mgmt
+// Copyright (C) 2013-2018+ James Shubin and the project contributors
+// Written by James Shubin <james@shubin.ca> and the project contributors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// run simple smoke tests to make sure most basic functionality works
+
+package integration_test
+
+import (
+	"log"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestHelp(t *testing.T) {
+	out, err := exec.Command(mgmt).Output()
+	if err != nil {
+		t.Fatal("Error: ", err)
+	}
+	if !strings.Contains(string(out), "--version, -v  print the version") {
```

Comment:
You sure we want to check this exact string? Is that API stable in the cli lib?

Reply:
Is anything stable at this point ;)

The test is easy enough modified if the cli lib API changes, also I don't expect it to change that often.